src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -26,6 +26,7 @@ $schedule-btn--height: 51px;
     background-color: opaque-black(0.9);
     height: $schedule-btn--height;
     padding-left: $NavBarSchedule--padding;
+    transition: all 400ms $transition-function;
```

Comment:
Thanks for that. Would be even better, is we had an extra class name for open buttons: { "NavBarWrapper__download-buttons--show": isVisible }, and give them the same `transform` properties as `.NavBarSchedule` has (to make them come together). 🙃

Reply:
fixed in c1e15fe93015c6a35048aac1213b8eddf3347c36